eagleproject/core/blockchain/const.py

```diff
@@ -29,6 +31,7 @@
     VAULT,
 )
 
+START_OF_EVERYTHING_TIME = datetime.strptime("18-9-2020", "%d-%m-%Y")
```

Comment:
Nit: If this is based on the timestamp below it, let's use the timestamp to set this datetime. That way there is one source of truth.

Reply:
The 1 line below it is the `block number` of the start of everything afaict.